tests/aws/services/cloudformation/resources/test_cloudwatch.py

```diff
@@ -43,24 +48,34 @@ def test_alarm_creation(deploy_cfn_template, snapshot):
     paths=[
         "$..StateReason",
         "$..StateReasonData",
-        "$..StateTransitionedTimestamp",
-        "$..StateValue",
-        "$..AlarmActions",
-        "$..AlarmRule",
-        "$..StateReason",
-        "$..StateReasonData",
-        "$..StateTransitionedTimestamp",
         "$..StateValue",
     ]
 )
 def test_composite_alarm_creation(aws_client, deploy_cfn_template, snapshot):
-    snapshot.add_transformer(snapshot.transform.cloudwatch_api())
+    snapshot.add_transformer(snapshot.transform.key_value("Region", "region-name-full"))
+
     stack = deploy_cfn_template(
         template_path=os.path.join(
             os.path.dirname(__file__), "../../../templates/cfn_cw_composite_alarm.yml"
         ),
     )
     composite_alarm_name = stack.outputs["CompositeAlarmName"]
+
+    def alarm_action_name_transformer(key: str, val: str):
+        if key == "AlarmActions" and isinstance(val, List) and len(val) == 1:
```

Comment:
I believe we should use the `list` type here and not from `typing`? 
```suggestion
        if key == "AlarmActions" and isinstance(val, list) and len(val) == 1:
```